python/taichi/lang/misc.py

```diff
@@ -528,7 +528,14 @@ def _block_dim(dim):
     get_runtime().prog.current_ast_builder().block_dim(dim)
 
 
-def loop_config(block_dim=None, serialize=None, parallelize=None):
+def loop_config(block_dim=None, serialize=False, parallelize=None):
+    """Sets directives for the next loop
+
+    Args:
+        block_dim (int): The number of threads in a block on GPU
```

Comment:
IIRC this is providing a list of config options, can we make this kwargs only by making it `*, block_dim=None, serialize=False,...`?